test_util/src/lib.rs

```diff
@@ -155,6 +156,22 @@ pub async fn run_all_servers() {
   let absolute_redirect_server_fut =
     warp::serve(routes).bind(([127, 0, 0, 1], REDIRECT_ABSOLUTE_PORT));
 
+  let routes = warp::path!("a" / "b" / "c")
```

Comment:
This is working fine, but I have less than 2 hours of rust coding in my life :stuck_out_tongue: , so feel free to improve it.

Reply:
can you say in words what the behavior of this is supposed to be? 

do none of the other servers cover this case?

(Just trying to push back on adding new servers here.)

Reply:
I wanted to, but with my limited rust knowledge was the best I could do. Probably can be added very easily to one of the other servers. 

Need the request to redirect to the same server either with: `some-path` or `/some-path`, without causing a redirect loop.

Reply:
@ry added the route to one of the other servers.

Reply:
nice!